### PR TITLE
Project settings

### PR DESCRIPTION
**Pull request by jackschonherr**

### CODE REVIEW

File: Consequences/Result.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿namespace Consequences;
+public struct Result<T> 
+{
+  private ResultItem<T>[] results;
```
Comment (Brennan1994) [inline Consequences/Result.cs:4]: results should probably be a readonly property Results. That way we can get the array of results out of the result object and add to it.

File: Consequences/ResultItem.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  private string resultName;
+  private T result;
+
+  public ResultItem(string name, T res)
+  {
+    resultName = name;
+    result = res;
+  }
+
+  public string Name()
```
Comment (Brennan1994) [inline Consequences/ResultItem.cs:13]: There is some fancy syntactic sugar to beautify what you've done here. Auto Properties are what they're called. Automatically implements the backing field and the accessors. I'll show these tomorrow,.

**Comment by Brennan1994** [comment]
We also addressed modifying the name-spacing. We'll make sure we push that before we merge, and the name of the branch got a little screwy with the rename. We can address that next go-around though

**Comment by Brennan1994** [comment]
https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/choosing-between-class-and-struct

**Review by HenryGeorgist — CHANGES_REQUESTED** [review]

**Review by Brennan1994 — APPROVED** [review]
Looks Good

**Pull request merged**